Utils/trigger_private_build.py

```diff
@@ -0,0 +1,162 @@
+import os
+import sys
+import json
+import time
+import argparse
+import requests
+import logging
+from typing import List
+import demisto_sdk.commands.common.tools as tools
+from Tests.scripts.utils.log_util import install_logging
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+PRIVATE_BUILD_INFRA_SCRIPTS = ['Tests/scripts/validate_premium_packs.sh', 'Tests/scripts/validate_premium_packs.py',
+                               'Tests/scripts/validate_index.py']
+PRIVATE_BUILD_INFRA_FOLDERS = ['Tests/private_build', 'Tests/Marketplace']
+
+TRIGGER_BUILD_URL = 'https://api.github.com/repos/demisto/content-private/dispatches'
+GET_DISPATCH_WORKFLOWS_URL = 'https://api.github.com/repos/demisto/content-private/actions/runs'
+WORKFLOW_HTML_URL = 'https://github.com/demisto/content-private/actions/runs'
+GET_WORKFLOW_URL = 'https://api.github.com/repos/demisto/content-private/actions/runs/{:s}/jobs'
+
+PRIVATE_REPO_WORKFLOW_ID_FILE = 'PRIVATE_REPO_WORKFLOW_ID.txt'
+
+GET_WORKFLOWS_MAX_RETRIES = 3
+
+GET_WORKFLOWS_TIMEOUT_THRESHOLD = 3600  # one hour
+
+
+def get_modified_files(branch_name: str = None) -> List[str]:
+    """ Gets modified files between master branch and the input branch_name, If the branch_name is empty the method
+        compare master branch with the commit sha1 from the environment variable CIRCLE_SHA1.
+
+    Args:
+        branch_name: The branch name to compare with master.
+
+    Returns: A list of modified files.
+
+    """
+    if not branch_name:
+        branch_name = os.environ.get('CIRCLE_SHA1')
+
+    files = []
+    files_string = tools.run_command(f'git diff --name-only origin/master...{branch_name}')
```

Comment:
One comment about that - this does not cover moving files from the private build paths to somewhere not in the paths, is that a case we want to cover? @guykeller or we can ignore?

Reply:
No need